openedx/core/djangoapps/contentserver/views.py

```diff
@@ -294,10 +294,7 @@ def load_asset_from_location(location):
     content = get_cached_content(location)
     if content is None:
         # Not in cache, so just try and load it from the asset manager.
-        try:
-            content = AssetManager.find(location, as_stream=True)
-        except (ItemNotFoundError, NotFoundError):  # lint-amnesty, pylint: disable=try-except-raise
-            raise
```

Comment:
What was this doing, and did you remove it because it should do the same with and without?

Reply:
Yeah, it's basically a no-op -- the [lint documentation](https://pylint.pycqa.org/en/latest/user_guide/messages/warning/try-except-raise.html) notes that a bare raise as the first line of a sole except block is pointless. (Raising something else, or having multiple except blocks with different behavior, or having other stuff before the raise -- those would not be covered by this lint.)

It [never](https://github.com/openedx/edx-platform/pull/11277/files#diff-da750c240a2307f59fdfbc3ab1fe6425bb299fbeefb58516571dd2420f4dfbc4R197) did anything other than this, so I think it was imitating some similar code that instead of raising would return a 404.